crates/driver/src/infra/api/mod.rs

```diff
@@ -39,6 +39,7 @@ impl Api {
         self,
         shutdown: impl Future<Output = ()> + Send + 'static,
         order_priority_strategies: Vec<OrderPriorityStrategy>,
+        settle_queue_size: usize,
```

Comment:
The settle queue size should be a per-driver configuration and does not need to show up in the API `serve()` function.